packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -240,6 +240,8 @@ export const start = async (zcf, privateArgs) => {
     getParamMgrRetriever,
     getInvitation: electorateParamManager.getInternalParamValue,
     getLimitedCreatorFacet: () => vaultFactory,
+    getGovernedApis: () => harden({}),
+    getGovernedApiNames: () => harden({}),
```

Comment:
I thought we said that undefined getters would be treated the same as getters returning empty. This is busywork for the contract author.

Reply:
This was clarified in chat: the auto-providing is done by `handleParamGovernance` which vaultFactory eschews (because it's an uncommonly complex governance)